cmakefiles/petsc/config/Debug.config.sh

```diff
@@ -1,17 +1,12 @@
 #!/bin/bash
-mkdir -p @PETSC_DIR@
-cp -r @INSTALL_DIR@/src/* @PETSC_DIR@
-cd @PETSC_DIR@
-
-# by default petsc will found default system libs such as blas lapack or openmpi
+unset MAKEFLAGS
 ./configure \
```

Comment:
This approach allows just add more CONFIGURE_OPTS, not bad, but there should be alternative to control more details, possibly set own script template. Then we must document which placeholders are supported.